### PR TITLE
Fonts: Add Google Noto Color Emoji font subset 

### PR DESCRIPTION
**Pull request by EchterAgo**
This will be bundled in the AppImage and loaded at runtime if the system lacks a usable emoji font.

### CODE REVIEW

**Comment by EchterAgo** [comment]
This also includes a small translation update for a string in the TX dialog.

File: gui/qt/main_window.py [hunk 1]
```diff
@@ -4307,7 +4307,7 @@ def cpfp(self, parent_tx, new_tx):
         vbox.addWidget(WWLabel(_(msg2)))
         grid = QGridLayout()
         grid.addWidget(QLabel(_('Total size') + ':'), 0, 0)
-        grid.addWidget(QLabel('%d bytes'% total_size), 0, 1)
+        grid.addWidget(QLabel(_(f'{total_size} bytes')), 0, 1)
```
Comment (cculianu) [inline gui/qt/main_window.py]: hehe i keep catching small stuff like this as i read code for cashaccts.. totally understand your nit here. :)

Comment (EchterAgo) [reply gui/qt/main_window.py]: I need to be able to translate "bytes" to "Bytes" for German :)

**Comment by cculianu** [comment]
This is awesome, Thank you 🙏

File: gui/qt/main_window.py (hunk 1 above)
Comment (cculianu) [reply gui/qt/main_window.py]: hehe :)

**Pull request merged**